pyperformance/data-files/benchmarks/bm_random/pyproject.toml

```
@@ -0,0 +1,10 @@
+[project]
+name = "pyperformance_bm_random"
+requires-python = ">=3.8"
```

[`random.binomialvariate`](https://docs.python.org/3/library/random.html#random.binomialvariate) is new in 3.12.

```suggestion
requires-python = ">=3.12"
```